tests/test_as_dict.py

```diff
@@ -0,0 +1,193 @@
+import click
+
+
+def test_export_dict():

```

Comment:
Split this into multiple tests for individual methods (`ParamType`, `Option`, `Command`, etc.).